tests/suites/refresh/refresh.sh

```diff
@@ -60,6 +60,28 @@ run_refresh_local() {
 	destroy_model "${model_name}"
 }
 
+run_refresh_local_resources() {
+	# Test a plain juju refresh with a local charm
+	echo
+
+	model_name="test-refresh-local-resources"
+	file="${TEST_DIR}/${model_name}.log"
+	charm_name="${TEST_DIR}/juju-qa-test.charm"
+
+	ensure "${model_name}" "${file}"
+
+	juju download juju-qa-test --no-progress - >"${charm_name}"
+	juju deploy "${charm_name}" juju-qa-test --resource foo-file="./tests/suites/resources/foo-file.txt"
+	wait_for "juju-qa-test" "$(idle_condition "juju-qa-test")"
+
+	juju refresh juju-qa-test --path "${charm_name}"
+
+	wait_for "juju-qa-test" "$(charm_rev "juju-qa-test" "1")"
+	wait_for "juju-qa-test" "$(idle_condition "juju-qa-test")"
```

Comment:
Do we need to check that the charm still has the expected resources after refresh